src/main/java/uk/gov/hmcts/reform/draftstore/DraftStoreApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableScheduling;
 
 @SpringBootApplication
+@EnableScheduling
 public class DraftStoreApplication {
```

Comment:
also I thought there was a need for a job scheduling component in reform ;)? 
We decided we aren't doing this?